app/src/main/java/com/imashnake/animite/features/home/Home.kt

```diff
@@ -126,7 +129,7 @@ fun Home(
                             verticalArrangement = Arrangement.spacedBy(dimensionResource(Res.dimen.large_padding))
                         ) {
                             HomeRow(
-                                list = trendingList,
+                                list = (trendingList as? Resource.Success)?.data?.media.orEmpty(),
```

Comment:
don't worry this won't stay as this, I'll end up changing the if statements / adding conditions to only show this composable when the list is not empty etc